lxd/events/events.go

```diff
@@ -37,12 +37,13 @@ func NewServer(debug bool, verbose bool) *Server {
 }
 
 // AddListener creates and returns a new event listener.
-func (s *Server) AddListener(group string, connection *websocket.Conn, messageTypes []string, location string, noForward bool) (*Listener, error) {
+func (s *Server) AddListener(group string, allGroups bool, connection *websocket.Conn, messageTypes []string, location string, noForward bool) (*Listener, error) {
```

Comment:
Please can you have this function return an error if `group` is non-empty and `allGroups` is true, as that should be an invalid combination.